generators/react-native/templates/app/modules/home/home-screen.js.ejs

```diff
@@ -20,13 +20,13 @@ function HomeScreen(props) {
         {account && account.login ? (
           <View style={[styles.authContainer, styles.authContainerTrue]} testID="authDisplayTrue">
             <Text style={styles.authText}>
-              <Ionicons name="md-checkmark-circle" size={22} color={'white'} /> You are signed in as {account.login}
+              <Ionicons name="checkmark-circle" size={22} color={'white'} /> You are signed in as {account.login}
```

Comment:
Icons don't render before this change. However, after it, they render kinda funky. I was unable to use CSS to fix this.

<table><tr><td>
<img src="https://github.com/jhipster/generator-jhipster-react-native/assets/17892/ee17827e-279b-4247-9b2c-a071a208d3bb">
</td>
<td><img src="https://github.com/jhipster/generator-jhipster-react-native/assets/17892/b7eefd66-19e3-4e6a-bf67-7c408af6a5c1">
</td>
</tr>
</table>

Reply:
I was able to fix this by adding the following CSS to each icon. This doesn't seem like a good fix.

```
style={{ position: 'absolute', marginLeft: -20 }}
```